tests/src/test/scala/com/nvidia/spark/rapids/SparkQueryCompareTestSuite.scala

```diff
@@ -577,6 +631,10 @@ trait SparkQueryCompareTestSuite extends FunSuite with Arm {
       qualifiers = qualifiers + s"NOT ON GPU[$execStr]"
     }
 
+    if (SparkSessionHolder.adaptiveQueryEnabled) {
```

Comment:
nit: I'm not a fan of a global, but I understand that it is the fastest way to plumb this through.